py/vtdb/field_types.py

```diff
@@ -93,7 +88,9 @@ def convert_bind_vars(bind_variables):
       new_vars[key] = times.DateTimeToString(val)
     elif isinstance(val, datetime.date):
       new_vars[key] = times.DateToString(val)
-    elif isinstance(val, (int, long, float, str, List, NoneType)):
+    elif isinstance(val, set):
```

Comment:
Yes because otherwise the bind var appears as set([1,23]) and gets sent as is, which doesn't get converted right at the protocol level and vttablet throws an error to the effect - invalid uin8[] arg ...

dbapi indicates expect an iterable type.